src/shared/logger.ts

```diff
@@ -95,7 +103,7 @@ export function getLogger(): Logger {
     if (defaultLogger) {
         return defaultLogger
     }
-    throw new Error ('Default Logger not initialized. Call logger.initialize() first.')
+    throw new Error('Default Logger not initialized. Call logger.initialize() first.')
```

Comment:
Will this message be surfaced to the user?

Reply:
Hopefully not. This will only turn up if:
* extension.ts isn't loaded, no other default logger has been initialized, and a logger call is made (in the case of tests)
* Someone attempts to log before this line: https://github.com/aws/aws-toolkit-vscode/blob/develop/src/extension.ts#L57